models/advanced-home-example/readme.md

```diff
@@ -0,0 +1,40 @@
+# Changes made between basic and advanced
+
+Semantic Types<br />
+Complex Types (Object)<br />
+Inheritance<br />
+Relationship properties<br />
+Non-targeted relationship<br />
+
+## Complex Type
+
+Added a new address complex type to collect address information on the ```Home``` model
+
+<img src="../../images/adv-home-object.png" style="max-width: 400px" />
```

Comment:
These images (which do look great btw) aren't very accessible to people using screenreaders. Consider adding alt text to the img tags that someone could use to piece together what part of what code file is being highlighted.